lib/tree-view.coffee

```diff
@@ -512,6 +506,20 @@ class TreeView extends View
       handle()
       handleError(error?.message)
 
+  showSelectedEntryInFileManager: ->
+    entry = @selectedEntry()
+    return unless entry
+
+    isFile = entry instanceof FileView
+    {command, args, label} = @fileManagerCommandForPath(entry.getPath(), isFile)
+    @openInFileManager(command, args, label, isFile)
+
+  showCurrentFileInFileManager: ->
+    return unless editor = atom.workspace.getActiveTextEditor()
+    return if editor.getPath() is undefined  # handle utitled tab
```

Comment:
Typo: `untitled`.  Does this also correctly handle "special" tabs like the Settings View?